test/test_utils/security.py

```diff
@@ -162,7 +162,14 @@ def __eq__(self, other):
             return test_utils.check_if_two_dictionaries_are_equal(
                 dataclasses.asdict(self),
                 dataclasses.asdict(other),
-                ignore_keys=["package_details", "title", "reason_to_ignore"],
+                ignore_keys=[
```

Comment:
can we avoid changing this behavior for all DLCs? perhaps using the utility function you just added to conditionally add more ignore statements for HF?

Reply:
Sure, I can update